x/gov/types/v1/params.go

```diff
@@ -68,7 +68,7 @@ func NewParams(
 	quorum, threshold, constitutionAmendmentQuorum, constitutionAmendmentThreshold, lawQuorum, lawThreshold, minInitialDepositRatio string,
 	burnProposalDeposit, burnVoteQuorum bool, minDepositRatio string,
 	quorumTimeout, maxVotingPeriodExtension time.Duration, quorumCheckCount uint64,
-	maxGovernors uint64, governorStatusChangePeriod time.Duration, minGovernorSelfDelegation string,
+	maxGovernors uint64 /*TODO remove maxGovernors? */, governorStatusChangePeriod time.Duration, minGovernorSelfDelegation string,
```

Comment:
I think it should be removed, I don't even know why it's still present in this branch

Reply:
done here [29c037e](https://github.com/giunatale/atomone/pull/3/commits/29c037e4af43e4103f7df374788894d734910ca9)